patches_for_WebRTC_org/m80/0007.2-Properly-handling-async-model-for-initializing-Media.patch

```diff
@@ -1,19 +1,34 @@
-From ee2a99a20bd6d53f0314a0303cff730391494ff9 Mon Sep 17 00:00:00 2001
+From c48576f03202179141476832cba6a76b4f4eddd7 Mon Sep 17 00:00:00 2001
 From: Augusto Righetto <aurighet@microsoft.com>
 Date: Wed, 27 May 2020 23:38:26 -0700
 Subject: [PATCH] Properly handling async model for initializing MediaCapture
  object
 
 Working around the stride and gap issues with NV12 memory layout
+
+Replacing error handling pattern from "Jump On Fail" to "Cascading ifs"
+    Removing THR and its goto
+
+Removing use of the ReleaseInterface function
+
+help_functions_winrt.cc without the sinful goto
+
+video_capture_winrt.cc deprived of all goto sins
+
+device_info_winrt, repent for the goto sins
+
+No more goto or THR sins
```

Comment:
nit: I'd stick to "Removed gotos from a, b, and c", this comes off as a bit snarky (not important if we don't plan to contribute this patch)

Reply:
Don't worry. :) Patch files are sanitized before contributing back. The process of contributing back involves applying the patch files to the WebRTC local repo, running the tidy apps and clean up log messages while rebasing.